composer.json

```diff
@@ -63,7 +63,9 @@
     "wpackagist-plugin/redis-cache": "*",
     "wpackagist-plugin/wordpress-seo": "*",
     "wpackagist-plugin/so-clean-up-wp-seo": "*",
-    "wpackagist-plugin/gravity-forms-wcag-20-form-fields": "*"
+    "wpackagist-plugin/gravity-forms-wcag-20-form-fields": "*",
+    "oscarotero/env": "^2.1",
+    "roots/wp-config": "^1.0"
```

Comment:
These should be on top of the requires right after `phpdotenv`. Otherwise, these are easily removed as a plugins.

Reply:
True. Going ahead and correcting this.